src/tree.ts

```diff
@@ -170,8 +176,12 @@ export class ProjectOutlineProvider implements vscode.TreeDataProvider<BaseNode>
         if (node) {
             return node.getChildren();
         }
-
-        return [this._currentConfigurationItem, this._currentBuildTargetItem, this._currentLaunchTargetItem];
+        if (configuration.getEnableLocalDebugRun()) {

```

Comment:
Since `enablement` does not work. I had to use this if. Could this be related to [this issue](https://github.com/microsoft/vscode/issues/102794)?

Reply:
I am not familiar with that vscode bug but thanks for finding the connection and commenting on the alternative. I agree we need to do hide/show the tree element via an IF in the code, until the "enablement" keyword will be properly implemented in VSCode for tree items.

Reply:
OK. I am glad you agree.